scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/core/rmproxies/RMProxiesManager.java

```diff
@@ -102,7 +102,7 @@ public RMProxiesManager(Credentials schedulerProxyCredentials) throws RMExceptio
      *
      * @param rmURI The URI of a started Resource Manager
      */
-    public abstract void rebindRMProxiesManager(final URI rmURI) throws RMException, RMProxyCreationException;
+    public abstract boolean rebindRMProxiesManager(final URI rmURI) throws RMException, RMProxyCreationException;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the redundant '!unknownSymbol!' thrown exception declaration(s). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)